include/zenoh_commons.h

```diff
@@ -279,7 +279,16 @@ typedef struct z_clock_t {
  * Options passed to the `z_close()` function.
  */
 typedef struct z_close_options_t {
-  uint8_t _dummy;
+  /**
+   * The timeout for close operation in milliseconds. 0 means default close timeout which is 10 seconds.
+   */
+  uint32_t timeout_ms;
```

Comment:
I'm wondering how to separate fields of options structures to "core" api and "zenoh-c" (or zenoh-pico) specific API.
It seems that if the structure contains zenoh-c specific fields, it should be `zc_close_options_t` itself.
This means that we have to add also the function `zc_close` which takes `zc_close_options_t` instead of common `z_close_options_t`.

Reply:
Well, `z_close` is not zenoh-c specific API, as well as option's fields like `timeout` and `out_concurrent` - they are present in Rust with some language-specific difference.

Otherwise following your logic, we should `zc_*` everything that has options API.....